lipo_app.py

```diff
@@ -0,0 +1,77 @@
+import streamlit as st
+import pandas as pd
+import torch
+from torch import nn
+from torchvision import transforms, models
+from PIL import Image
+import altair as alt
+from collections import OrderedDict
+import gc
+
+st.title('Lipohypertrophy Prediction')
+
+#load the model
+cnn_model = models.densenet121(pretrained=True)
+new_layers = nn.Sequential(OrderedDict([
+            ('new1', nn.Linear(1024, 500)),
+            ('relu', nn.ReLU()),
+            ('new2', nn.Linear(500, 1))
+        ]))
+cnn_model.classifier = new_layers
+
+cnn_model.load_state_dict(torch.load('densemodels.pth', map_location=torch.device('cpu'))) #put the directory here where cnn_model.pt is located
```

Comment:
Update this model to the final model on Onedrive!!

Reply:
Will do!!